vioscsi/helper.c

```diff
@@ -120,13 +120,16 @@ ENTER_FN_SRB();
             virtqueue_notify(adaptExt->vq[QueueNumber]);
         }
     } else {
+        // virtqueue_add_buf() returned -28 (ENOSPC), i.e. no space for buffer, or some other error
```

Comment:
See my comment above. ENOSPC means no disk space. ENOBUFS means no buffers and ENOMEM is can't alloc mem